frontend/src/pages/MeetingRoom/MeetingRoom.spec.tsx

```diff
@@ -184,6 +184,17 @@ describe('MeetingRoom', () => {
     expect(screen.getByTestId('publisher-container')).toBeInTheDocument();
   });
 
+  it('should display spinner until session is connected', () => {
+    sessionContext.connected = false;
+    publisherContext.publisher = mockPublisher;
+    const { rerender } = render(<MeetingRoomWithProviders />);
+    rerender(<MeetingRoomWithProviders />);
+    sessionContext.connected = true;
+    expect(screen.getByTestId('progress-spinner')).toBeInTheDocument();
```

Comment:
Nit, could we change the order of these two lines and add a line break? I know we're not doing calling a rerender, but setting connected to true and testing if we see the spinner is a little unintuitive.

Reply:
Yep, makes sense!